src/api/pastel-rpc/network-stats/network-stats.ts

```diff
@@ -0,0 +1,313 @@
+import { rpc, TRPCConfig } from '../rpc'
+import {
+  TAddressDelta,
+  TAddressesBalance,
+  TAddressMempool,
+  TAddressUtxo,
+  TBlockInfo,
+  TBlockSubsidy,
+  TChainTips,
+  TGetaddressbalance,
+  TGetaddressdeltas,
+  TGetaddressmempool,
+  TGetaddresstxids,
+  TGetaddressutxos,
+  TGetbestblockhash,
+  TGetblock,
+  TGetblockcount,
+  TGetblockhash,
+  TGetblocksubsidy,
+  TGetchaintips,
+  TGetdifficulty,
+  TGetmempoolinfo,
+  TGetmininginfo,
+  TGetNetTotals,
+  TGetnetworkhashps,
+  TGetNetworkinfo,
+  TGetrawmempool,
+  TGetrawtransaction,
+  TGettotalbalance,
+  TGettransaction,
+  TGettxout,
+  TGettxoutsetinfo,
+  TGetwalletinfo,
+  Tlistaddresses,
+  TListReceivedByAddress,
+  TListTransactions,
+  Tlisttransactions,
+  TListUnspent,
+  Tlistunspent,
+  TMempoolInfo,
+  TMiningInfo,
+  TNetTotals,
+  TNetworkInfo,
+  TRawMempoolInfo,
+  TRawTransaction,
+  TTotalBalance,
+  TTransactionInfo,
+  TTxout,
+  TTxoutsetInfo,
+  TValidateaddress,
+  TValidateaddresses,
+  TWalletInfo,
+  TZListReceivedByAddress,
+} from './type'
+
+type TStatistic = {
+  hashrate: number
+  difficulty: number
+}
+
+export async function getStatisticInfo(
```

Comment:
All functions in this file are just wrappers of the `rpc` method. Please remove them